src/sql-parser/src/keywords.rs

```diff
@@ -263,6 +263,7 @@ define_keywords!(
     JSONB,
     KAFKA,
     KEY,
+    KEY_TYPE,
```

Comment:
Adding `KEY_TYPE` and `VALUE_TYPE` as keywords should no longer be necessary!

Reply:
Ah - thank you!